chart/templates/deployment.yaml

```diff
@@ -75,7 +75,7 @@ spec:
       serviceAccountName: {{ include "cf-service-operator.fullname" . }}
       automountServiceAccountToken: true
       containers:
-      - name: controller
+      - name: cf-service-operator
```

Comment:
in _helpers.tpl, there is already a logic using cf-service-operator.name,
e.g. for setting a label
  app.kubernetes.io/name: {{ include "cf-service-operator.name" . }}
we could just leverage that one

Reply:
absolutely. Thxs for the proposal.